tests/www/views/test_views_tasks.py

```diff
@@ -754,3 +754,69 @@ def test_set_task_instance_action_permission_denied(session, client_ti_without_d
         follow_redirects=True,
     )
     check_content_in_response(expected_message, resp)
+
+
+@pytest.mark.parametrize(
+    "task_search_tuples",
+    [
+        (
+            [
+                ("example_xcom", "bash_push"),
+                ("example_bash_operator", "run_this_last"),
+            ]
+        ),
+        (
+            [
+                ("example_subdag_operator", "some-other-task"),
+            ]
+        ),
```

Comment:
```suggestion
        [("example_xcom", "bash_push"), ("example_bash_operator", "run_this_last")],
        [("example_subdag_operator", "some-other-task")]
```

nit, one less layer of nesting and easier to read.

Reply:
Good suggestion, I go crazy with the indents sometimes! I'll update now.